.gitignore

```diff
@@ -6,6 +6,7 @@ setup_env.csh
 setup_env.sh
 *.swp
 *.orig
+*.rstx
```

Comment:
Out of curiosity, what's the reason for the *.rstx? I am not familiar with this file type.

Reply:
It is part of the documentation. It appears to be an output of the .rst compile.